src/main/java/ua/lviv/iot/novaPoshtaAPI/controller/CourierController.java

```diff
@@ -0,0 +1,40 @@
+package ua.lviv.iot.novaPoshtaAPI.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import ua.lviv.iot.novaPoshtaAPI.model.Courier;
+import ua.lviv.iot.novaPoshtaAPI.service.CourierService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/couriers")
+public class CourierController {
```

Comment:
don't see the API to create Couriers or update their data

Reply:
Couriers can only be created and updated via departments, so there won't be any "hanging" couriers without a department